README.md

```diff
@@ -126,6 +116,7 @@ const { parseArgs } = require('@pkgjs/parseargs');
 // type:string & multiple
 const args = ['-f', '--foo=a', '--foo', 'b'];
 const options = {
+  f: { type: 'boolean' },
   foo: {
     type: 'string',
     multiple: true,
```

Comment:
Same here, remove the funky `f` and trim down to the example of interest.

```
// type:string & multiple
const options = {
  foo: {
    type: 'string',
    multiple: true,
  },
};
const args = ['--foo=a', '--foo', 'b'];
const { values, positionals } = parseArgs({ args, options });
// values = { foo: [ 'a', 'b' ] }
// positionals = []
```

Reply:
* https://github.com/pkgjs/parseargs/pull/74/commits/9b76d55779372e20cddbc67284787a4283f9a14f
* https://github.com/pkgjs/parseargs/pull/74/commits/e56246c59179b3e1ca38654cde68636c05ac6c0e